### PR TITLE
get_place_functionality_and_unit_test

### PR DESCRIPTION
**Pull request by RoxVer**
Method for getting one place created as well as unit test for it.

### CODE REVIEW

**Comment by ghost** [comment]
reviewed

File: src/main/java/ua/softserve/rv_018/greentourism/service/PlaceServiceImpl.java [hunk 1]
```diff
@@ -42,6 +42,17 @@
...
+	public Place findOne(int id) {
+		logger.info("> Place findOne id:{}", id);
+
+        Place place = placeRepository.findById(id);
+
+        logger.info("< Place findOne id:{}", id);
+
+        return place;
+	}
+
```
Comment (Bmaks-github) [inline src/main/java/ua/softserve/rv_018/greentourism/service/PlaceServiceImpl.java:54]: Please, format the code. Make indents.

File: src/main/webapp/WEB-INF/views/main.jsp [hunk 2]
```diff
@@ -15,10 +15,13 @@
 	<link rel="stylesheet" href="https://npmcdn.com/leaflet@1.0.0-rc.3/dist/leaflet.css">
 	<!-- Custom styles -->
 	<link rel="stylesheet" href="${pageContext.request.contextPath}/resources/app/css/style.css">
+		<!-- Place page styles -->
+	<link rel="stylesheet" href="${pageContext.request.contextPath}/resources/app/css/place.css">
```
Comment (Bmaks-github) [inline src/main/webapp/WEB-INF/views/main.jsp:19]: Put all your css files in style.css.

File: src/main/webapp/WEB-INF/views/main.jsp [hunk 3]
```diff
@@ -15,10 +15,13 @@
...
 	<!-- Supporting mobile devices -->
 	<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
 </head>
 <body>
+<img src="<c:url value="/images/gps-icon.png" />" />
```
Comment (Bmaks-github) [inline src/main/webapp/WEB-INF/views/main.jsp:24]: Please, no hard-coding. Use angularJS directives. In that case use ng-src.

File: src/main/webapp/resources/app/components/place/place.component.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+						$scope.findById = function() {
+							// update only if id chosen
+							if (!$scope.id) {
+								return;
+							}
+							$scope.errorMessage = "";
+
+							var successCallBack = function(response) {
+								// console.log(response);
```
Comment (Bmaks-github) [inline src/main/webapp/resources/app/components/place/place.component.js:19]: Delete unnecessary comments.

**Pull request closed without merging**